src/udev/udev-builtin-net_id.c

```diff
@@ -284,7 +291,7 @@ static bool is_pci_ari_enabled(struct udev_device *dev) {
 }
 
 static int dev_pci_slot(struct udev_device *dev, struct netnames *names) {
```

Comment:
The dev_port is also taken into account in `dev_pci_onboard()`, but I never heard of a board that actually features a built-in InfiniBand HCA, so decided not to touch it.